stactools_pipelines/pipelines/gedi_calval_copc/config.yaml

```diff
@@ -0,0 +1,7 @@
+---
+  id: "gedi-calval-copc"
+  compute: "awslambda"
+  secret_arn: "arn:aws:secretsmanager:us-west-2:916098889494:secret:MAAP-STAC-auth-dev/MAAP-workflows-EsykqB" # cognito secret ARN
+  ingestor_url: "https://stac-ingestor.maap-project.org"
```

Comment:
suggestion : change this to the 'test' ingestor (that ingests to the test catalog) for a first run to check if things work properly (maybe also shorten the csv for that test, since you don't need all the data to be ingested for the test)

Reply:
Good point! I'll make those changes